api/src/models/users.py

```diff
@@ -0,0 +1,42 @@
+from db import db
+
+class Users(db.Model):
+
+    __tablename__ = 'Users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(120), nullable=False)
+    is_bot = db.Column(db.Boolean, nullable=False)
+
+    def save(self):
+
+        try:
+            db.session.add(self)
+            db.session.commit()
+        except Exception as e:
+            return
```

Comment:
This needs to be handled, logged at least.

Reply:
fixed